gridstatus/ieso.py

```diff
@@ -2454,3 +2458,683 @@ def _parse_intertie_schedule_flow_file(
+        intertie_prices = root.findall(".//IntertieLMPrice", ns)
+
+        for intertie_price in intertie_prices:
+            location = intertie_price.find("IntertiePLName", ns).text
+
+            components = intertie_price.findall("Components", ns)
+
+            lmp_values = {}
+            loss_values = {}
+            energy_congestion_values = {}
+            external_congestion_values = {}
+            # Net Interchange Scheduling Limit
+            nisl_values = {}
+
+            for component in components:
+                component_type = component.find("LMPComponent", ns).text
+                intervals = component.findall("IntervalLMP", ns)
+
+                for interval in intervals:
+                    interval_num = interval.find("Interval", ns).text
+                    lmp_value_elem = interval.find("LMP", ns)
+
+                    if (
+                        lmp_value_elem is None
+                        or lmp_value_elem.text is None
+                        or lmp_value_elem.text.strip() == ""
+                    ):
+                        continue
+
+                    lmp_value = float(lmp_value_elem.text)
+
+                    if component_type == "Intertie LMP":
+                        lmp_values[interval_num] = lmp_value
+                    elif component_type == "Energy Loss Price":
+                        loss_values[interval_num] = lmp_value
+                    elif component_type == "Energy Congestion Price":
+                        energy_congestion_values[interval_num] = lmp_value
+                    elif component_type == "External Congestion Price":
+                        external_congestion_values[interval_num] = lmp_value
+                    elif (
+                        component_type
+                        == "Net Interchange Scheduling Limit (NISL) Price"
+                    ):
+                        nisl_values[interval_num] = lmp_value
+
+            for interval_num in lmp_values.keys():
+                if (
+                    interval_num in loss_values
+                    and interval_num in energy_congestion_values
+                    and interval_num in external_congestion_values
+                    and interval_num in nisl_values
+                ):
+                    interval_start = base_datetime + pd.Timedelta(
+                        minutes=(int(interval_num) - 1) * 5,
+                    )
+                    interval_end = interval_start + pd.Timedelta(minutes=5)
+
+                    lmp = lmp_values[interval_num]
+                    congestion = energy_congestion_values[interval_num]
+                    loss = loss_values[interval_num]
+                    external_congestion = external_congestion_values[interval_num]
+                    nisl_value = nisl_values[interval_num]
+
+                    # Note that inertie LMP includes external congestion and NISL
+                    energy = lmp - congestion - loss - external_congestion - nisl_value
```

Comment:
@cwaldoch found this is the definition of energy for the intertie data.

<img width="667" alt="Screenshot 2025-04-23 at 17 12 53" src="https://github.com/user-attachments/assets/335bf8fc-e813-4d62-b922-3d987a6cb03e" />